components/_patterns/02-molecules/card/_card.scss

```diff
@@ -3,3 +3,11 @@
   background-color: $gray-lightest;
   padding: 1em;
 }
+
+.card__sub {
+  margin-bottom: 1em;
+  
+  p {
```

Comment:
This should be a class... and shouldn't be nested... what's the thought process here?